x/pairing/keeper/unstaking.go

```diff
@@ -52,13 +52,22 @@ func (k Keeper) UnstakeEntry(ctx sdk.Context, provider bool, chainID string, cre
 	if existingEntry.Deadline < holdBlocks {
 		existingEntry.Deadline = holdBlocks
 	}
+	details := map[string]string{"stake_entry": existingEntry.String()}
+	utils.LogLavaEvent(ctx, logger, types.UnstakeCommitNewEventName(provider), details, " Appending new unstake entry (removing stake entry).")
```

Comment:
you added an event for each time you use this function, we dont need another one inside, you can maybe send one event from this function and add description string as a parameter for the function

Reply:
done